graylog2-server/src/main/java/org/graylog2/inputs/transports/HttpPollTransport.java

```diff
@@ -30,10 +30,7 @@
 import org.graylog2.plugin.ServerStatus;
 import org.graylog2.plugin.configuration.Configuration;
 import org.graylog2.plugin.configuration.ConfigurationRequest;
-import org.graylog2.plugin.configuration.fields.ConfigurationField;
-import org.graylog2.plugin.configuration.fields.DropdownField;
-import org.graylog2.plugin.configuration.fields.NumberField;
-import org.graylog2.plugin.configuration.fields.TextField;
+import org.graylog2.plugin.configuration.fields.*;
```

Comment:
`*` imports are generally not used in Graylog projects. As a general standard, we always import specifically only the needed classes. This can be done by setting the 'Class count to use import with '*'' to a very high value as described in this article:

https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-